moveit_ros/planning_interface/test/subframes_test.cpp

```diff
@@ -87,12 +84,11 @@ bool moveToCartPose(geometry_msgs::PoseStamped pose, moveit::planning_interface:
 // The box spawns in front of the gripper, the cylinder at the tip of the gripper, as if it had been grasped.
 void spawnCollisionObjects(moveit::planning_interface::PlanningSceneInterface& planning_scene_interface)
 {
-  const std::string log_name = "sapwn_collision_objects";
+  const std::string log_name = "spawn_collision_objects";
   double z_offset_box = .25;  // The z-axis points away from the gripper
-  double z_offset_cylinder = .12;
+  double z_offset_cylinder = .1;
```

Comment:
I got the test to pass by putting this back.  0.1 puts the cylinder inside the eff.

Reply:
This is weird. The cylinder is "far" away from the eef and changing the position by 2cm shouldn't be a problem in any case. Could you double-check please?